### PR TITLE
Bump to GMT 6.2.0

### PR DESCRIPTION
**Pull request by weiji14**
Release on https://github.com/GenericMappingTools/gmt/releases/tag/6.2.0.

### CODE REVIEW

**Comment by seisman** [comment]
The CI fails because it tries to validate the GenericMappingTools/try-gmt repository, rather than the pull request repository.

Could you please try changing the `github.repository` to `github.event.repository.name` at the following line and see if it works?

https://github.com/GenericMappingTools/try-gmt/blob/2983ba2993586967c459ced69aa7b26f230a5320/.github/workflows/repo2docker.yml#L32

File: .github/workflows/repo2docker.yml [hunk 1]
```diff
@@ -29,4 +29,4 @@ jobs:
 
       - name: Run repo2docker
         shell: bash -l {0}
-        run: jupyter-repo2docker --no-run --ref ${{ github.event.pull_request.head.sha }} https://github.com/${{ github.repository }}
+        run: jupyter-repo2docker --no-run --ref ${{ github.event.pull_request.head.sha }} https://github.com/${{ github.event.repository.name }}
```
Comment (weiji14) [inline .github/workflows/repo2docker.yml:32]: Now it's trying to use https://github.com/try-gmt instead :frowning_face:

Comment (seisman) [reply .github/workflows/repo2docker.yml:32]: Try `github.event.pull_request.head.repo.full_name`? See https://github.community/t/get-the-head-repository-name-in-prs/17669

Comment (weiji14) [reply .github/workflows/repo2docker.yml:32]: Ah cool, was trying to search for it. xref docs at https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request.

```suggestion
        run: jupyter-repo2docker --no-run --ref ${{ github.event.pull_request.head.sha }} https://github.com/${{ github.event.pull_request.head.repo.full_name }}
```

**Review by seisman — APPROVED** [review]

**Comment by weiji14** [comment]
Oops, merged too soon. Forgot to update the GMT version on the main README.md :sweat_smile:

**Comment by seisman** [comment]
😅 Sorry for that, please open a separate PR.

**Comment by weiji14** [comment]
Ok, can I get added as a maintainer on this try-gmt repo? Will be easier to create a branch directly.

**Comment by seisman** [comment]
Sure, you're added as a maintainer now.

**Pull request merged**